src/js/apps/forms/form/form_app.js

```diff
@@ -110,6 +115,7 @@ export default App.extend({
     'submit': 'onFormServiceSubmit',
     'success': 'onFormServiceSuccess',
     'ready': 'onFormServiceReady',
+    'submissionUpdated': 'onFormServiceSubmissionUpdated',
```

Comment:
The event shouldn't be past-tense, and should generally be `context:verb:subject`.  So in this case `update:submission` probably.